src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -153,23 +153,25 @@ const ReviewPage = (props: IReviewPageProps) => {
         </>
       )}
 
-      <Row>
-        <Col>
-          {Object.values(rootReview.children)
-            .sort(reviewLevelSortMethod)
-            .map(reviewLevel => (
-              <ReviewCollapsible
-                splitView={splitView}
-                hugoSymbol={props.hugoSymbol}
-                isGermline={props.isGermline}
-                key={reviewLevel.currentValPath}
-                baseReviewLevel={reviewLevel}
-                handleAccept={props.acceptReviewChangeHandler}
-                handleDelete={props.rejectReviewChangeHandler}
-              />
-            ))}
-        </Col>
-      </Row>
+      {rootReview ? (
+        <Row>
+          <Col>
+            {Object.values(rootReview.children)
```

Comment:
The sort with the filter can be costly with memory if there's a lot of data in the object. Are you expecting to have a lot of data being processed here?

Reply:
never mind I forgot sort returns the same array